mobly/test_runner.py

```diff
@@ -472,7 +421,7 @@ def run_test_class(self, test_cls_name, test_cases=None):
                 self.results += e.results
                 raise e
 
-    def run(self, test_class=None):
+    def run(self, test_classes):
```

Comment:
How come we were ok with a single test_class before, but now we need a list? Is there a usecase?